Makefile

```diff
@@ -1011,6 +1011,10 @@ ci-unit-nightly:
 .ci-prepare-integration:
 	sudo -E ./scripts/travis/prepare-integration.sh
 
+.PHONY: .ci-travis-permissions-workaround
```

Comment:
Instead of adding another make target for specifically running on Travis, could you simply add this to the `scripts/travis/prepare-integration.sh` script called from the `.ci-prepare-integration` target?

Reply:
I will go back and look at why kami added this to see if that is enough. I cherry-picked this change from #4772

Reply:
Seems it was for packs and unit tests as well, not just integration tests (which had the fix first).

Reply:
Are you referring to tests for packs in StackStorm-Exchange? Because they exclusively use CircleCI, not Travis, and I don't believe ST2 used in those tests runs this make target.

Maybe we should just rename that script, update any references to it, and update the make target name as well. 😆

Reply:
https://github.com/StackStorm/st2/compare/8c55ab37bc1dd546558aafa5c9f277c2430f2c7c..d19dc8dbda30a4cedadb89a699cf064396e0e405

make targets:
ci-py3-unitci-packs-test

Reply:
These make targets are used in the 3rd and 4th travis environments:
- Lint Checks, Packs Tests (Python 2.7)
- Unit Tests, Pack Tests (Python 3.6)

Plus the 2nd and 5th travis environments which already had the workaround:
- Integration Tests (Python 2.7)
- Integration Tests (Python 3.6)

Leaving the 1st environment as the only one not using the workaround:
- Unit Tests (Python 2.7 MongoDB 4.0)